### DISCUSSION TITLE
xpressLRS and EP1 RX connected but no servos moments no radio inputs in INAV!

### DISCUSSION THREAD

**Discussion by flyingdragonozzy** [ExpressLRS/ExpressLRS#847] [Q&A]
I have F405 WSE FC and tx16s latest frimware 

I also flashed the Inav to the latest 3.1 and the both RX and TX were properly installed and I have solid yellow lED in the RX and the radio was showing perfect connection with the Inav and all the servos were working perfectly 

but today I connected the FC to the Inav and I was not able to see any movements in Inav and the servos not working at all! 

the telemetry is connected and the yellow lED in the RX is solid 

the back LED in the TX is solid Red 

I noticed the TX is very HOT to the touch! I have no fan installed yet! 

do you think the TX is burned ?

I falshed the Inav and the FC again and calibrated the FC 

all the setup are in green and I have no red except the fail safe is red! 

I also noticed that FC  sometimes is not welling to connect until I re open new Inav configuration
and unplug the FC and re connect again 

sometimes takes few times to connect!!!!! 

thanks for any help!

**Comment by avsaase** [comment]
Do you see movement of the channels in the Receiver tab? If so, then I'd say this problem is on the INAV side, not ExpressLRS.

**Comment by flyingdragonozzy** [comment]
there is no movement in the Receiver tab in the Inav configuration software.
no movements in the airplane servos too! 

my radio TX have red solid LED and my RX EP1 have solid yellow LED! 
thank you

**Reply by palgy** [reply]
I have the same issue ... I have telemetry but i don't have any sticks movement ...

**Reply by palgy** [reply]
I managed to solve the issue by replacing the receiver ... so it's can be something with the receiver or something with the connection ( soldering or cables  )

**Reply by rwindsorjr** [reply]
Got the same problem on 2 different EP1 s  from different places. On a Maytek F405-wing. Tried three different Uarts. 1 2 and 6 no joy . Didnt try with the shipped firmware.  0ne on 1,0,1 and the other 1.1.0 . No reaction on INAV receiver but the RM TX16S has a telemetry connection.  Tomorrow I will go back as far as I can .

**Comment by estebanavv** [comment]
It seems that there is an ongoing Issue with happymodel's EP1 EP2 the behavior is erratic and in most cases fixed itself changing versions
#780 
#720 
#655 
#643
#597
https://www.rcgroups.com/forums/showpost.php?p=47745249&postcount=425

**Comment by estebanavv** [comment]
I flashed 1.0x Mainttenance https://github.com/ExpressLRS/ExpressLRS/commits/1.0.x-maintenance and the issue with 2 EP1s were solved.

![image](https://user-images.githubusercontent.com/9424792/132067227-ae1b3a3e-6627-4e6b-9d38-a3416ffc1be6.png)
